### DISCUSSION TITLE
Project name suggestions

### DISCUSSION THREAD

**Discussion by ffenix113** [ffenix113/zigbee_home#2] [Q&A]
Welcome everyone!

I think this project may have a better name, which would be more descriptive, or at least more interesting.
A play on "Nordic" or it's theme is preferable for my taste.

If you have an idea for the name - put your suggestions below!

**Comment by Hedda** [comment]
Some name suggestions:

* BeeHome (an amalgamation of the words "Zig**Bee**" and "**Home**"))
* ZigHome (other alternative amalgamation of the words "**Zig**Bee" and "**Home**")
* ZDHome (abbreviation of "Zigbee Device Home")
* ZCLHome? ("Zigbee Cluster Library Home")
* ZAPHome (abbreviation of "ZCL Advanced Platform Home", name idea comming from Matter project's "ZAP" project)
    * For reference also see -> https://github.com/ffenix113/zigbee_home/discussions/4 

Idea if you could get the zigpy organization to adopt this project and you become a member then could maybe rename it "zigpy-home" 😉 

https://github.com/zigpy

https://github.com/zigpy/zigpy

zigpy organization is an informal group of open-source developers that maintain the zigpy library and related open-source Zigbee projects that are used in the [ZHA integration component for Home Assistant](https://www.home-assistant.io/integrations/zha/), the [Zigbee Plugin for Domoticz](https://www.domoticz.com/wiki/ZigbeeForDomoticz), and the [Zigbee Plugin for Jeedom](https://doc.jeedom.com/en_US/plugins/automation%20protocol/zigbee/) (competing open-source home automation software). 

PS:  FYI, very recently a member of the zigpy organization, (puddly), became a full-time employe of Nabu Casa (who has several full-time employees that together work on many open-source projects related to Home Assistant (including ESPHome, ZHA/zigpy, and Z-Wave-JS).

https://www.nabucasa.com/about/

**Reply by Hedda** [reply]
If want a more descriptive name when perhaps include words like "firmware" and "open-source", or "free", "libre", "FOSS" "FLOSS"  😀 

* BeeFree
* BeeFreeHome
* BeeHome
* BeeHomeFree
* ZigFree
* ZigFreeHome
* LibreZig
* FreeZig
* BeeFirmware
* ZigFirmware
* ZCLFirmware
* ZigFOSS
* ZigFLOSS
* BeeOpen
* OpenBee
* ZigOpen
* OpenZig
* BeeLibre
* LibreBee
* ZigLibre
* LibreZig

**Reply by ffenix113** [reply]
Thank you for the suggestions, those are nice ones!

Let's keep them as an option for now.

**Comment by Hedda** [comment]
Bump this discussion about changing the project name, as ”Zigbee Home” is definitivtly not a good project name considering that it is not only too generic but due to the fact that ”Zigbee Home Automation” (or ”ZHA” and ”Zigbee HA” for short) is the name of one of the application profiles for devices from the Zigbee specifications, and it is just one of the many available Zigbee application profiles).

Searching for ”Zigbee Home” on Google is way to general so gives too many result instead of a specific project, including finding all devices that uses Zigbee Home Automation profile, such as these -> https://csa-iot.org/csa-iot_products/?p_keywords=&p_type%5B%5D=17&p_type%5B%5D=14&p_type%5B%5D=1053&p_program_type%5B%5D=955&p_certificate=&p_family=

PS: This name is even double confusing becuse the native Zigbee Gateway implementation built-into Home Assistent is also called ”Zigbee Home Automation” and ”ZHA”, which I too pointed out is a poor choice of name for the exact same reason as stated above -> https://www.home-assistant.io/integrations/zha/

**Reply by ffenix113** [reply]
I 100% agree with you. But it also not easy to come up with a good name.. I do like `OpenBee` that you mentioned earlier, and to piggy back on that - `NordicBee`. I believe latter name could have some nice icons :slightly_smiling_face: .

I appreciate your suggestions, they are really valuable to me, but I don't think that at this point of time there is a definite name for the project.

**Reply by Hedda** [reply]
EasyBee or BeeEasy could be two additional alternative names to OpenBee and BeeOpen, as in it will be Easy and Zigbee combined.

Regardless, a manufacturer specific name like ”NordicBee” would probably not good in the long term if ever port to other chips.

ESPHome has for example already been ported to Raspberry Pi’s RP2040 and is being ported to Nordic Semi’s nRF52/nRF53.
